resources/js/src/app/helper/StickyElement.js

```diff
@@ -12,6 +13,12 @@ const STICKY_EVENTS = [
     "move-sticky"
 ];
 
+const STICKY_EVENTS_PASSIVE = [
+    "scroll",
+    "touchstart",
+    "touchmove"
```

Comment:
Man kann hier auch direkt noch "wheel" mit aufnehmen.

Reply:
wheel ist doch gar nicht in der Non-Passive Liste?

Reply:
das ist nicht in der Liste von den Events auf die gehorcht wird:

```
const STICKY_EVENTS = [
    "resize",
    "scroll",
    "touchstart",
    "touchmove",
    "touchend",
    "pageshow",
    "load",
    "move-sticky"
];
```

Reply:
Nevermind then